front/app/components/PostShowComponents/Comments/ChildCommentForm.tsx

```diff
@@ -92,7 +89,8 @@ const CancelButton = styled(Button)`
 `;
 
 interface Props {
-  postId: string;
+  ideaId?: string;
+  initiativeId?: string;
```

Comment:
Replaced postId everywhere with these. It's risky because both are undefined, not hard to miss one, but I see it as a first step towards creating intermediary components that deal with idea/initiative specific logic. postId is confusing, since you don't know which one it is about.

But not sure if that split will be part of the work now, because it can turn out to be a lot.